### PR TITLE
migrated to swift 4.0

### PR DESCRIPTION
**Pull request by andresinaka**

### CODE REVIEW

**Comment by andresinaka** [comment]
@volbap / @agerace  do you want to review and merge this??

File: EventSource/EventSource.swift [hunk 1]
```diff
@@ -47,15 +47,12 @@ open class EventSource: NSObject, URLSessionDataDelegate {
         self.receivedString = nil
         self.receivedDataBuffer = NSMutableData()
 
-
-        var port = ""
-        if let optionalPort = self.url.port {
-            port = String(optionalPort)
-        }
+        let port = String(self.url.port ?? 80)
```
Comment (agerace) [inline EventSource/EventSource.swift:50]: Shouldn't it be port 8080 instead of just 80?

Comment (andresinaka) [reply EventSource/EventSource.swift:50]: I couldn't find a standard port for event source but as it's just an http request I think the right port would be 80 if not specified.

Comment (agerace) [reply EventSource/EventSource.swift:50]: Got it!

**Pull request merged**